force-app/main/default/triggers/CellPINTrigger.trigger

```diff
@@ -0,0 +1,8 @@
+trigger CellPINTrigger on Cell__c (before update) {
```

Comment:
We already have trigger on Cell object. Use existing [CellTrigger](https://github.com/msenyk/sf-practicum-2024/blob/main/force-app/main/default/triggers/CellTrigger.trigger).

Reply:
The trigger for Pin generation has been rewritten in CellTrigger